### ISSUE TITLE
[Bug] - Wrong colors if there is extends or super keywro inside generic of generic

### ISSUE THREAD

**Issue by hetacz** [dinbtechit/vscode-theme#132]
## Describe the bug

Create generic of generic, that uses super or exends keyword, like this:
`private static final Set<Class<? extends Throwable>> RETRYABLE_EXCEPTIONS = Set.of(TimeoutException.class);`
![image](https://github.com/dinbtechit/vscode-theme/assets/82400620/33d88651-b23a-4fd9-968b-16ff91a33af3)

**extends or super keyword is highlighted in green instead of blue**

## Platform & Language Details 
- Language: Java
- IDE Details: 

```
IntelliJ IDEA 2023.2.2 (Ultimate Edition)
Build #IU-232.9921.47, built on September 12, 2023
Licensed to Rafał Michalski
Subscription is active until May 15, 2024.
Runtime version: 17.0.8+7-b1000.22 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 8
Registry:
    debugger.new.tool.window.layout=true
    documentation.show.toolbar=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0

Non-Bundled Plugins:
    com.github.leomillon.uuidgenerator (4.5.1)
    Batch Scripts Support (1.0.13)
    lermitage.intellij.extratci (1.27.0)
    com.nasller.CodeGlancePro (1.7.5)
    bundled-intellij-idea-help (232.9921.47)
    com.github.copilot (1.3.0.3323)
    com.godwin.json.parser (1.7)
    nl.bryanderidder.regexrenamefiles (1.2.5)
    net.seesharpsoft.intellij.plugins.file-preview (1.6.4)
    com.alayouni.ansiHighlight (23.2.0)
    com.robohorse.robopojogenerator (2.4.1)
    org.intellij.RegexpTester (2.2.0)
    com.intellij.grazie.pro (0.3.254)
    com.tobszarny.activetabhighlighter.plugin.id (1.5.1)
    com.crunch42.openapi (1.66)
    com.jetbrains.edu (2023.9-2023.2-696)
    com.andrey4623.rainbowcsv (2.0.2)
    com.intellij.ideolog (222.1.0.0)
    Statistic (4.2.7)
    com.eversql.plugins.jetbrains (1.3.1)
    com.intellij.properties.bundle.editor (232.8660.88)
    Key Promoter X (2023.2.0)
    lermitage.intellij.extra.icons (2023.2.11-231)
    com.intellij.kubernetes (232.9921.86)
    net.ashald.envfile (3.4.1)
    com.intellij.datagen (232.8660.88)
    com.intellij.spring.websocket (232.8660.142)
    String Manipulation (9.11.3)
    test-rail-plugin (1.6.2)
    CheckStyle-IDEA (5.83.0)
    com.intellij.plugin.adernov.powershell (2.2.0)
    com.intellij.aqua (232.9921.28)
    tarn.gradle.dependencies (1.6.3)
    com.github.camork.fileExpander (2.2)
    com.alibaba.autonavi.qa.testng (1.0)
    org.jetbrains.jumpToLine (0.1.13)
    me.prouge.sealedfluentbuilder (0.5)
    cn.renyuzhuo.plugin.easygradle (1.0.1)
    com.jetbrains.packagesearch.intellij-plugin (232.9921.28)
    com.intellij.plugin.poe (0.4.1)
    MavenRunHelper (4.26.0-IJ2022.2)
    com.intellij.aspectj (232.8660.142)
    com.github.setial (4.1.0)
    GrepConsole (12.22.1-IJ2021.1)
    net.seesharpsoft.intellij.plugins.csv (3.2.1-232)
    com.intellij.ml.llm (232.9826)
    com.xiyuan.TypeScriptExecutor (1.1.0 for idea2021.2)
    allure-ee-idea (3.19.1)
    dev.eltonsandre.intellij.spring.assistant.plugin (2023.0.1)
    com.haulmont.jpab (2023.3.5-232)
    ru.adelf.idea.dotenv (2023.2)
    com.clutcher.comments_highlighter (2023.7.13)
    com.intellij.tide (232.8660.88)
    com.github.dinbtechit.vscodetheme (1.10.3)
    izhangzhihao.rainbow.brackets (2023.3.5)
    com.mallowigi.colorHighlighter (17.1.0)
    mobi.hsz.idea.gitignore (4.5.1)
    zielu.gittoolbox (500.0.21+222)

Kotlin: 232-1.9.0-IJ9921.47
 ```

**Comment by dinbtechit** [comment]
@hetacz  - Thank you for reporting this bug. Hopefully we can patch this in the next release. 🙌

**Comment by dinbtechit** [comment]
Fixed in [v1.10.4](https://github.com/dinbtechit/vscode-theme/releases/tag/v1.10.4)
